website/content/en/docs/concepts/nodeclasses.md

```diff
@@ -431,6 +431,8 @@ spec:
 
 AMI Selector Terms are used to configure custom AMIs for Karpenter to use, where the AMIs are discovered through ids, owners, name, and [tags](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html). **When you specify `amiSelectorTerms`, you fully override the default AMIs that are selected on by your EC2NodeClass [`amiFamily`]({{< ref "#specamifamily" >}}).**
 
+Keep in mind that amiFamily determines the bootstrapping mode, while amiSelectorTerms specifies the specific AMI to be used.Therefore, ensure consistency between amiFamily and amiSelectorTerms to avoid conflicts.
```

Comment:
```suggestion
Note that [`amiFamily`]({{< ref "#specamifamily" >}}) determines the bootstrapping mode, while amiSelectorTerms specifies specific AMIs to be used. Therefore, you need to ensure consistency between [`amiFamily`]({{< ref "#specamifamily" >}}) and amiSelectorTerms to avoid conflicts during bootstrapping.
```

Just some thoughts around wording. Can you make this change across the `preview` path instead of the `docs` path? I admit that it's confusing, but we pull the `preview` path through each time we do a new release. `docs` is effectively a symlink to the the current release version `v0.36`. If you wanted this to be in the current version, you'd need this change in both `v0.36` and `docs`

Reply:
This PR would be preferred.